docs/account-management/checking-the-connection-between-github-and-semaphore-2.0.md

```diff
@@ -13,28 +13,62 @@ This web page shows you:
 - Organizations that you are a member of, and the level of access that Semaphore
   2.0 has to these organizations.
 
-**If the page is not showing Semaphore 2.0 application on GitHub, and you
+If the page is not showing Semaphore 2.0 application on GitHub, and you
 previously logged in to Semaphore 2.0 with the GitHub account that you are
-currently using, then please contact Semaphore support.**
+currently using, please check your [account settings](https://me.semaphoreci.com/account) page.  
+Under "Repository access" sections, click on grayed out checkmarks to reauthorize Semaphore app.
+
+**If [Semaphore 2.0 application page on GitHub](https://github.com/settings/connections/applications/328c742132e5407abd7d) is still not showing Semaphore 2.0 application
+please contact Semaphore support.**
 
 
 ## Grant additional permissions
 
 When you sign up with your GitHub account, Semaphore is asking only for your
 email. If you want to grant additional permissions, that is to let Semaphore
 2.0 access public and/or private repositories that you have access to, please
-visit:
+visit [account settings](https://me.semaphoreci.com/account) page.
 
-```
-https://{ORGANIZATION_NAME}.semaphoreci.com/projects/scopes/github
-```
+To grant Semaphore 2.0 access to public or private repositories click on grayed out checkmark to go through authorization process.
```

Comment:
To grant Semaphore 2.0 access to public or private repositories, click on the grayed out checkmark to go through authorization process.